changelog.dd

```diff
@@ -73,6 +82,86 @@ assert(m.front[1] == "12");
 -------
 )
 
+$(LI $(LNAME2 optimization, `findLocalMin` was added to `std.numeric`.)
+    $(P $(REF findLocalMin, std,numeric) finds a real minimum of a real function `f(x)` via bracketing.)
+-------
+import std.numeric, std.math;
+
+auto ret = findLocalMin((double x) => (x-4)^^2, -1e7, 1e7);
+
+assert(ret.x.approxEqual(4.0));
+assert(ret.y.approxEqual(0.0));
+-------
+)
+
+$(LI $(LNAME2 slice_ptr, `ptr` property and public constructor were added to
+    `std.experimental.ndslice.slice.Slice`.)
+    $(P `ptr` property allows to access `Slice`'s
+        underlaying pointer or range. It is suggested to read about
+        $(REF Slice, std,experimental,ndslice,slice)
+        internal binary representation before using the property.
+        `ptr` property is used in $(LINK2 https://github.com/libmir/mir, Mir)
+        for $(LINK2 http://docs.mir.dlang.io/latest/mir_sparse_package.html, sparse tensors).
+        `ndslice` developer mirror in Mir will be removed as
+        soon as LDC (LLVM D compiler) support DMDFE 2.072.
+    ) 
+    $(P Public constructor for `Slice` was added to support
+        $(STDMODREF ndslice, std.experimental.ndslice) integration
+        with other languages and libraries such as Julia language and NumPy library.
+    )
+)
+
+$(LI $(LNAME2 slice_alloc, `slice`, `shape`, `ndarray`, and other utilities
+    were added to `std.experimental.ndslice.slice`.)
+    $(P 
+        $(REF shape, std,experimental,ndslice,slice)
+        $(REF slice, std,experimental,ndslice,slice),
+        $(REF makeSlice, std,experimental,ndslice,slice),
+        $(REF ndarray, std,experimental,ndslice,slice), and
+        $(REF makeNdarray, std,experimental,ndslice,slice)
+        allocation utilities were added to $(STDMODREF ndslice.slice, std.experimental.ndslice)
+        )
+    $(P Example: Transposing common 2D array using `ndslice`)
+-----
+import std.experimental.ndslice;
+
+auto ar = [[0, 1, 2], [3, 4, 5]];
+
+auto sh = ar.shape;         // [2, 3] type of size_t[2]
+auto sl = slice!int(sh);    // allocates slice with corresponding shape
```

Comment:
oh sorry - that's right, but then we could fix that? 
Or is this problematic because `shape` will throw an exception?